src/Kliva/Views/MainPage.xaml.cs

```diff
@@ -24,7 +24,6 @@ public MainPage()
         protected override void OnNavigatedTo(NavigationEventArgs e)
         {
             base.OnNavigatedTo(e);
-            UpdateVisualState(VisualStateGroup.CurrentState);
             _activityList.SelectedIndex = -1;
```

Comment:
Seems that with selectionmode changing on the activitylist in this PR we would no longer need this selectedindex hack? Can we verify this?